learningresources/tests/base.py

```diff
@@ -157,3 +157,14 @@ def get_course_single_tarball(self):
         """
         path = join(abspath(dirname(__file__)), "testdata", "courses")
         return self.copy_file(join(path, "single.tgz"))
+
+    def get_course_static_tarball(self):
+        """
+        Get the path to a course with course.xml in the root of the
+        archive and a static folder with two assets (one of which is
+        in a subfolder.
+
+        Returns: path (unicode): absolute path to tarball.
```

Comment:
Can you change this to:

    Returns:
        unicode: absolute path to tarball

It's nitpicky but if we ever need automatic docstring inspection for something it'll be useful for these to conform strictly to the spec